File: workspace/3_Taken/src/taken/Periode.java
package taken;

import java.util.Date;

//zie leesmij.txt
public class Periode {

	private Date start;
	private Date eind;

	public Periode(Date start, Date eind) {
		super();
		this.start = start;
		this.eind = eind;
	}

	public Date getStart() {
		return start;
	}

	public void setStart(Date start) {
		this.start = start;
	}

	public Date getEind() {
		return eind;
	}

	public void setEind(Date eind) {
		this.eind = eind;
	}

	@Override
	public String toString() {
		return "Periode [start=" + start + ", eind=" + eind + "]";
	}

}

File: solution/3_Taken/3_Taken/src/Periode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace taken
{
    /// <summary>
    /// zie leesmij.txt
    /// </summary>
    public class Periode
    {
        private DateTime? start;
        private DateTime? eind;

        public Periode(DateTime? start, DateTime? eind) : base()
        {
		    this.start = start;
		    this.eind = eind;
	    }

        public DateTime? getStart()
        {
		    return start;
	    }

        public void setStart(DateTime? start)
        {
		    this.start = start;
	    }

        public DateTime? getEind()
        {
		    return eind;
	    }

        public void setEind(DateTime? eind)
        {
		    this.eind = eind;
	    }

	    public String toString() {
		    return "Periode [start=" + start + ", eind=" + eind + "]";
	    }
    }
}
